.github/workflows/deploy.yml

```
@@ -0,0 +1,24 @@
+name: Deploy blog
```

```suggestion
name: Build blog
```

Protože deploy je jen někdy, ale build vždycky?

---

Hmm, vždycky je jen lint, nevím jestli Danube Delta umí zvlášť build.

---

Hmm, vidím jenom preview, ale to se bude snažit otevřít prohlížeč si myslím https://github.com/honzajavorek/danube-delta/#usage

---

Každopádně build je vždycky, protože když neprojde tak zčervená pull request. Deploy je jen pro začleněné změny.